### DISCUSSION TITLE
Updating a docker based UVDesk v1.0.17 to the Latest

### DISCUSSION THREAD

**Discussion by gehric** [uvdesk/community-skeleton#667] [Q&A]
Hi All, I am struggling to update a docker based UVDesk v1.0.17 to the latest version without losing any data, has anyone succeeded in doing this

**Comment by PeopleInside** [comment]
Hi, thank you for your question. Maybe UVdesk staff can help you on the Docker update procedure.
I don't use Docker, I have installed UVdesk by composer or FTP.

In any case let me suggest and reminder to take a full all file and database backup before do any try to upgrade, so if something went wrong you can restore the backup.

**Comment by gehric** [comment]
I did ask the support staff, they suggest that i come here too see if anyone has a solution

**Reply by PeopleInside** [reply]
Where did you asked?
What kind of issue do you have upgrading with docker?
Maybe if you still have issue you can try to upgrade by to upgrade using FTP method.
https://forums.uvdesk.com/topic/1833/how-to-upgrade-uvdesk

Don't forget to backup all files and database before proceed.
